lib/src/validation-server/spots/validate.ts

```diff
@@ -0,0 +1,38 @@
+import { body, endpoint, Integer, request, response, String } from "../../lib";
+import { HttpMethod } from "../../models/http";
+import { Headers, UnprocessableEntityError } from "./utils";
+
+@endpoint({
+  method: "POST",
+  path: "/validate"
+})
+export class Validate {
+  @request
+  request(@body body: ValidateRequest) {}
+
+  @response({ status: 200 })
+  response(@body body: ValidateResponse) {}
+
+  @response({ status: 422 })
+  unprocessableEntityError(@body body: UnprocessableEntityError) {}
+}
+
+export interface ValidateRequest {
+  request: {
+    method: HttpMethod;
+    path: String;
+    headers: Headers;
+    body: String;
+  };
+  response: {
+    status: {
+      code: Integer;
+    };
```

Comment:
doesn't look like there's any benefit from wrapping `code` in a `status` object.

Reply:
I agree, will remove